index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="test.js"></script>
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+<body>
+    <h1>Web Calculator</h1>
+    <div id="inputBox">
+      <input type="number" id="firstNumber" />
+      <span>+</span>
+      <input name="first" id="secondNumber" />
+      <span>=</span>
+      <input type="button" value="Submit" style="margin: auto;" onclick="addNumbers()" />
```

Comment:
Try not to do any in-line styling at this stage. Its fine for now, but moving forward, create a CSS file to separate these concerns (the structure that is HTML and the styling that is CSS)